src/main/java/com/example/tweet/controllers/TopController.java

```diff
@@ -30,25 +32,44 @@ public ModelAndView topTweetPageString() {
     }
 
     @PostMapping("/insert")
-    public ModelAndView InsertTweet(HttpServletRequest request) {
+    public String InsertTweet(HttpServletRequest request, RedirectAttributes redirectAttributes) {
         int userId = Integer.parseInt(request.getParameter("user_id"));
         String content = (String) request.getParameter("content");
 
         int resultNum = tweetService.insertTweet(userId, content);
         System.out.println(resultNum + "件のTweetを追加");
-        // 遷移先画面を設定
-        ModelAndView model = new ModelAndView("/top.html");
-        return model;
+
+        return "redirect:/";
     }
 
     @PostMapping("/delete")
-    public ModelAndView deleteTweet(HttpServletRequest request) {
+    public String deleteTweet(HttpServletRequest request, RedirectAttributes redirectAttributes) {
         int id = Integer.parseInt(request.getParameter("id"));
         // 削除ボタン押下されたTweetのIDを利用して削除
         int resultNum = tweetService.deleteTweet(id);
         System.out.println(resultNum + "件のTweetを削除");
-        // 遷移先画面を設定
-        ModelAndView model = new ModelAndView("/top.html");
+
+        return "redirect:/";
+    }
+
+    @GetMapping("/edit{id}")
+    public ModelAndView editProcess(HttpServletRequest request) {
+        ModelAndView model = new ModelAndView("/edit.html");
+        int tweetId = Integer.parseInt(request.getParameter("id"));
+        // 編集ボタン押下されたTweetのIDを利用して削除
+        Tweet editTargetTweet = tweetService.selectTweetById(tweetId);
+
+        model.addObject("tweet", editTargetTweet);
         return model;
     }
```

Comment:
```suggestion
    public String editProcess(HttpServletRequest request, Model model) {
        final int tweetId = Integer.parseInt(request.getParameter("id"));
        // 編集ボタン押下されたTweetのIDを利用して削除
        final Tweet editTargetTweet = tweetService.selectTweetById(tweetId);

        model.addAttribute("tweet", editTargetTweet);
        return "edit";
    }
```
戻り値全部Stringで統一できて見通しがいいので、Model使う実装で良いように思います。

Reply:
戻り値の型をStringに変更しました。
[d8274f1](https://github.com/i-v-FSD/springTraining/pull/1/commits/d8274f1885ab272694953ba98cb3baff3c709778)